analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ConditionEvaluatesToConstantTest.cs

```diff
@@ -26,26 +26,26 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ConditionEvaluatesToConstantTest
     {
-        private readonly VerifierBuilder sonarVerifier = new VerifierBuilder<SymbolicExecutionRunner>().WithBasePath(@"SymbolicExecution\Sonar")
+        private readonly VerifierBuilder sonar = new VerifierBuilder<SymbolicExecutionRunner>().WithBasePath(@"SymbolicExecution\Sonar")
```

Comment:
I think we can find a more descriptive name than this. 

Calls like `sonar.AddPaths(` do not make sense to me and I prefer the `sonarVerifier` between these two.

Reply:
When working on any part of code related to SE, it is a necessary prerequisite to understand that there are two engines. One is called `Sonar`, the other one is called `Roslyn`. It comes from their root difference and this naming is used everywhere around the SE topic.

So following the same strategy in UTs makes sense. 

This discussion should be happening on `NullPointerDereferenceTest.cs` case. In every other test, it's just a preparation for the future. I can rename these preparations to `builder` like other tests, but it doesn't make much sense. We need to rename them from their current value so it makes sense to rename them to their final value. So we don't need to re-do that multiple times.

Reply:
While I understand that there are 2 engines and one of them is ours, my argument was that `sonar` is too generic and `sonar.AddPaths(` looks confusing.

I personally don't like this and prefer the old name.

Reply:
We're in rule UTs and we know what kind of code to expect here - that there's a verifier builder, we'll configure it and use it.

Having this concrete example and mandatory knowledge that there are two engines, this piece of code is clear:
```C#
        [TestMethod]
        public void NullPointerDereference_Sonar_CS() =>
            sonar.AddPaths("NullPointerDereference.cs").WithConcurrentAnalysis(false).Verify();

        [TestMethod]
        public void NullPointerDereference_Roslyn_CS() =>
            roslynCS.AddPaths("NullPointerDereference.cs").WithConcurrentAnalysis(false).Verify();
```

Reply:
I think it is clear from text, especially considering that `.AddPaths` is used in a similar context a lot of time in the code base. However, in most cases the `VerifierBuilder` property contains the term `builder`. So unless we don't plan to change it throughout the whole code base, I'd prefer buying a bit more verbose for consistency's sake. Looking at other uses `builderSonar` would be more in line than `sonarVerifier`, though.

Reply:
I don't have a strong opinion on this change but we had a similar discussion recently and also dopped the "builder" part.
https://github.com/SonarSource/sonar-dotnet/pull/6719#discussion_r1096079139
In my opinion neither "verifier" nor "builder" carry much information for the reader and can safely be dropped.

Reply:
Ok, I think it's hard to find a consensus for this. I will approve the PR just to avoid blocking things. @pavel-mikula-sonarsource, it's up to you if you want to consult the team and get a more descriptive name for this.